pkg/apis/externaldns/types.go

```diff
@@ -86,7 +86,7 @@ type Config struct {
 	AWSZoneTagFilter                   []string
 	AWSAssumeRole                      string
 	AWSProfiles                        []string
-	AWSAssumeRoleExternalID            string
+	AWSAssumeRoleExternalID            string `secure:"yes"`
```

Comment:
why do we need this?
I guess we do not need this, so please cleanup

Reply:
The label is needed to mask the `AWSAssumeRoleExternalID` value in the logs.
![image](https://github.com/user-attachments/assets/0aaab490-d0b0-4643-bf32-ea5e8c455a0f)

Reply:
relevant code:
- https://github.com/kubernetes-sigs/external-dns/blob/f5f7d23d869a7c0a5bdde198a6322ee83bc10beb/pkg/apis/externaldns/types.go#L37-L39
- https://github.com/kubernetes-sigs/external-dns/blob/f5f7d23d869a7c0a5bdde198a6322ee83bc10beb/pkg/apis/externaldns/types.go#L375-L394